---
source: packages/nemo-frontend/tests/type_errors.rs
expression: "type_errors::render_ty_error(&source, &err, false)"
input_file: packages/nemo-frontend/test_data/failing/unknown_field.nemo
---
[13] Error: Unknown field. F does not have a field named y
   ╭─[source:2:2]
   │
 6 │   F { x = 10 }.y
   │                ┬  
   │                ╰── Unknown field 'y' for struct 'F'
───╯

